src/deploymentSlotActions.ts

```diff
@@ -54,7 +54,12 @@ class SwapStep extends WizardStep {
 
     async prompt(): Promise<void> {
         const deploymentSlots: DeploymentSlotNode[] = await this.slot.getParentNode<DeploymentSlotsNode>().getChildren();
-        let otherSlots: QuickPickItemWithData<null>[] = [];
+        let otherSlots: QuickPickItemWithData<null>[] = [{
+            label: 'production',

```

Comment:
Does the portal use lower case 'p'? I'd prefer uppercase, but I'm fine if that's what the portal does

Reply:
Portal uses "production". Since the slot name is part of the site URL, although case insensitive, it is usually all lower case.